pwiz_tools/Skyline/Executables/BuildMethod/BuildThermoMethod/Program.cs

```diff
@@ -169,6 +172,12 @@ public static ListItem FromLine(int lineNum, string[] fields, Dictionary<int, st
                 if (curHeader.Equals("compound", StringComparison.InvariantCultureIgnoreCase))
                 {
                     item.Compound = curValue;
+                    var match = Regex.Match(curValue, @".+\(([\+|-])\)?(\d+)\).*");

```

Comment:
Consider whether code would be simplified by using Stellar isolation list. Otherwise, comment this better to explain what is being done. And probably explain that this EXE uses the Fusion isolation list for the Stellar. (confusing)